maestro/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "maestro.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "maestro.name" . }}
+    helm.sh/chart: {{ include "maestro.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "maestro.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "maestro.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: error-logs
```

Comment:
If put the logs on stdout, would it be easier to be monitored

Reply:
these are different from the normal app logs.
This will be audited and reviewed by humans, should be persistent (if pod dies or moved to another node, evicted, etc), currently we still don't have logs aggregation/ ELK or any other way to persist logs, so I had to do this for now until we decide on another way to store these failure events.